apps/prairielearn/src/middlewares/authzCourseOrInstance.ts

```diff
@@ -32,6 +43,23 @@ function clearOverrideCookies(res: Response, overrides: Override[]) {
   });
 }
 
+const SelectAuthzDataSchema = z.object({
+  mode: EnumModeSchema,
```

Comment:
This mode may come from `authz_data`, which means that it could technically not follow the schema (in which case that is a bad thing). `coalesce($req_mode, access_mode.mode) AS mode,`

Reply:
I think this is fine, given no errors in tests.